mkdocs.yml

```diff
@@ -34,6 +34,7 @@ nav:
       - Available Applications: basics/available-apps.md
       - How to Migrate to DS: basics/migration.md
       - Frequently Asked Questions: basics/faq.md
+      - Support: basics/support.md
```

Comment:
Your branch is missing recent changes. The format of the navigation has changed. You'll want to update your local branch by pulling the latest changes. I also recommend using pull bot https://github.com/apps/pull

Reply:
i'll update... or should it have now that i installed pull bot the other day?